src/components/organisms/Navbar/NavbarAction/NavbarAction.md

```diff
@@ -0,0 +1,21 @@
+### Summary
+
+[`<NavbarAction />`](#/Components/Organisms/Navbar/NavbarSignIn) renders the navbar CTA. On Desktop its a [`<Link />`](#/Components/Atoms/Link) styled as a primary button. On Mobile it renders a [user icon](https://fontawesome.com/icons/user?style=solid). You can update the CTA text on desktop
+
+### Examples
+
+- Default
+
+```ts { "props": { "style": { "backgroundColor": "#00B9A7" } } }
```

Comment:
can you change the `backgroundColor` to white here? below too

Reply:
I've set it to teal as the icon is white and when on mobile can't see the icon